server/db/controllers/proposals.js

```diff
@@ -47,8 +47,8 @@ async function get(req, res) {
 /**
  * Add a Proposal
  */
-export function add(req, res) {
-  if (!req.user) return res.sendStatus(401);
+export function add(req, res) {  
+  if (!req.user || !eventConfig.cfp) return res.sendStatus(401);
```

Comment:
so add is also update, right? 
If so, then you need to allow `isReversimTeamMember` here event if `eventConfig.cfp` is false

Reply:
no, there is an update method

```javascript
/**
 * Update a proposal
 */
async function update(req, res) {
...
```